darwin/client.py

```diff
@@ -104,18 +103,18 @@ def _get_headers(self, refresh=False):
         if refresh:
             return {
                 "Content-Type": "application/json",
-                "Authorization": f"Bearer {self._refresh_token}",
+                "Authorization": f"Bearer {self.refresh_token}",
             }
         else:
-            return {"Content-Type": "application/json", "Authorization": f"Bearer {self._token}"}
+            return {"Content-Type": "application/json", "Authorization": f"Bearer {self.token}"}
 
-    def get(self, endpoint: str, retry: bool = True, raw: bool = False):
+    def get(self, endpoint: str, retry: bool = False, raw: bool = False):
```

Comment:
I think the `retry` value should be `True` by default, since that's probably the most common and desirable use case of it